godtools/App/Features/UserLessonProgress/Presentation/ResumeLessonProgressModal/ResumeLessonProgressModal.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ResumeLessonProgressModal.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/29/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResumeLessonProgressModal: View {
+    
+    private let buttonHeight: CGFloat = 48
+    private let modalInset: CGFloat = 28
+    private let buttonInset: CGFloat = 20
+    private let buttonSpace: CGFloat = 12
+
+    @ObservedObject private var viewModel: ResumeLessonProgressModalViewModel
+    
+    init(viewModel: ResumeLessonProgressModalViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            let totalSpaceAroundModal = modalInset * 2
+            let modalWidth = geometry.size.width - totalSpaceAroundModal
+            let totalSpaceAroundButtons = (buttonInset * 2) + buttonSpace
+            let buttonWidth = (modalWidth - totalSpaceAroundButtons) / 2
+            
+            ZStack {
+                if #available(iOS 15.0, *) {
```

Comment:
Also @levieggertcru, do you think we can bump the minimum deployment target to 15?

Reply:
We got the go ahead to bump to iOS 15 so feel free to make that change in this PR.